### PR TITLE
Update metadata schema

### PR DESCRIPTION
**Pull request by yinlinchen**
Resolve: https://webapps.es.vt.edu/jira/browse/LIBTD-1169

1. Arrange field order same as [Geoblacklight's default schema](https://github.com/geoblacklight/geoblacklight/blob/master/schema/geoblacklight-schema.md)
2. Remove DEPRECATED field
3. Update field

### CODE REVIEW

File: lib/geoblacklight/dataingest.rb [hunk 1]
```diff
@@ -16,14 +16,12 @@ class DataIngest
     "dc:creator": {required: true},
     "dc:language": {required: true},
     "dc:publisher": {required: true},
-    "dc:relation": {required: false},
     "dc:type": {required: true},
     "dct:spatial": {required: true},
     "dct:temporal": {required: false},
     "dct:issued": {required: false},
     "ispartof": {required: true},
     "georss:box": {required: true},
```
Comment (tingtingjh) [inline lib/geoblacklight/dataingest.rb]: I saw you made changes on line 76. But should this "georss:box" field be completely replaced by "solr_gem"?

File: lib/geoblacklight/dataingest.rb [hunk 2]
```diff
@@ -62,35 +60,35 @@ def solrdatahash(content)
...
       rec['solr_geom'] = "ENVELOPE("+ gdata[1] + "," + gdata[3] + "," + gdata[2] + "," + gdata[0] + ")"
     end
-    rec['dc_title_s'] = content[17]
-    rec['dc_description_s'] = content[18]
-    rec['dc_format_s'] = content[19]
-    rec['dc_subject_sm'] = content[20]
+    rec['dc_creator_sm'] = content[7]
+    rec['dc_description_s'] = content[8]
+    rec['dc_format_s'] = content[9]
+    rec['dc_language_s'] = content[10]
```
Comment (tingtingjh) [inline lib/geoblacklight/dataingest.rb]: 'dc_language_s' or 'dc_language_sm'?

File: lib/geoblacklight/dataingest.rb [hunk 3]
```diff
@@ -62,35 +60,35 @@ def solrdatahash(content)
...
+    rec['dc_publisher_s'] = content[11]
```
Comment (tingtingjh) [inline lib/geoblacklight/dataingest.rb]: Similarly 'dc_publisher_sm'?

File: lib/geoblacklight/dataingest.rb [hunk 4]
```diff
@@ -131,10 +129,10 @@ def validaterecord(row)
           result += "dc_identifier field is not a valid URL. "
       
         elsif item == "georss:box" && (!content.respond_to?("split") || content.split(",").length != 4)
```
Comment (tingtingjh) [inline lib/geoblacklight/dataingest.rb]: "solr_gem" here?

File: lib/geoblacklight/dataingest.rb (hunk 1 above)
Comment (tingtingjh) [reply lib/geoblacklight/dataingest.rb]: Is "dc:type" optional now?

File: lib/geoblacklight/dataingest.rb [hunk 5]
```diff
@@ -16,14 +16,12 @@ class DataIngest
     "dc:creator": {required: true},
```
Comment (tingtingjh) [inline lib/geoblacklight/dataingest.rb:16]: creator, description, format, language, publisher,, ispartof,  spatial, layer:geom_type and layer_id optional?

Comment (yinlinchen) [reply lib/geoblacklight/dataingest.rb:16]: Although these fields are not required in the geoblacklight solr schema, these are required fields for GeoData.

**Review by tingtingjh — APPROVED** [review]
The order of the fields is correctly rearranged according to columns of the test CSV file. I was able to see the metadata of the testing datasets being populated as expected.

**Pull request merged**